diff --git a/api/utils/getImage.js b/api/utils/getImage.js
index 12bc1a8fe664f77614d46df0619724239c07af60..d00b8bf96b2521018085579d1550ccb4d70e43d9 100644
--- a/api/utils/getImage.js
+++ b/api/utils/getImage.js
@@ -1,4 +1,5 @@
 // @ts-check
+import fs from "node:fs";
 import crypto from "node:crypto";
 import objectHash from "object-hash";
 import getImageSources from "./getImageSources.js";
@@ -20,6 +21,14 @@ export default async function ({
   artDirectives,
   transformConfigs,
 }) {
+  {
+    if (!fs.existsSync("./srcs.txt")) {
+      fs.writeFileSync("./srcs.txt", "");
+    }
+
+    fs.appendFileSync("./srcs.txt", src + "\n");
+  }
+
   const args = Array.from(arguments);
 
   const hash = objectHash(args);
