diff --git a/components/Icon.astro b/components/Icon.astro
index f0a768219271862b2d774b49ccb0e1405a600834..68df026e82002377ebc605ad564e0ae59193db4d 100644
--- a/components/Icon.astro
+++ b/components/Icon.astro
@@ -107,7 +107,11 @@ if (props.size) {
   props.width = props.size;
   props.height = props.size;
   delete props.size;
+} else {
+  props.width ??= null;
+  props.height ??= null;
 }
+
 const renderData = iconToSVG(iconData, { width: "auto", height: "auto" });
 const normalizedProps = { ...renderData.attributes, ...props };
 const normalizedBody = renderData.body;
